pkg/cmd/grafana-cli/commands/datamigrations/to_unified_storage.go

```diff
@@ -75,6 +76,7 @@ func ToUnifiedStorage(c utils.CommandLine, cfg *setting.Cfg, sqlStore db.DB) err
 		provisioning,
 		nil, // no librarypanels.Service
 		sort.ProvideService(),
+		&actest.FakeAccessControl{ExpectedEvaluate: true}, // can only be run by an admin using grafana-cli
```

Comment:
Looking into a better way of doing this, but the rest of the PR is ready for review